src/astx/tools/transpilers/python.py

```diff
@@ -444,6 +444,12 @@ def visit(self, node: astx.WhileStmt) -> str:
         body = self._generate_block(node.body)
         return f"while {condition}:\n{body}"
 
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.YieldExpr) -> str:
+        """Handle YieldExpr nodes."""
+        value = self.visit(node.value) if node.value else ""
+        return f"yield {value}"
```

Comment:
please use **strip**, so if the value is empty, it will remove the extra empty space at the end of the string.

```suggestion
        return f"yield {value}".strip()
```